x-pack/plugins/ml/public/application/timeseriesexplorer/components/series_controls/series_controls.tsx

```diff
@@ -24,7 +24,7 @@ import type {
 } from '../../../../../common/types/anomaly_detection_jobs';
 import { useMlKibana } from '../../../contexts/kibana';
 import { APP_STATE_ACTION } from '../../timeseriesexplorer_constants';
-import type { ComboBoxOption, EntityControlProps } from '../entity_control/entity_control';
+import type { ComboBoxOption, Entity, EntityControlProps } from '../entity_control/entity_control';
```

Comment:
As discussed offline, is there a quick way to get the 'delete selection' button into the control? Otherwise with both a 'by' and 'partition' field, I'm finding it quite difficult to return the controls to the default empty selections.

Reply:
Enabled 'delete selection' button 👍 
![image](https://github.com/user-attachments/assets/b3105319-72f9-4cb2-8091-fd6c3c3fe59d)